#!/usr/bin/env python
PACKAGE = "two_d_guidance"
    
from dynamic_reconfigure.parameter_generator_catkin import *
    
gen = ParameterGenerator()

# Pure Pursuit
traffic_light_enum = gen.enum([ gen.const("red",    int_t, 0, "Red"),
                                gen.const("amber",  int_t, 1, "Amber"),
                                gen.const("green",  int_t, 2, "Green")],
                              "traffic_light_mode")
gen.add("traffic_light", int_t, 0, "traffic light", 0, 0, 2, edit_method=traffic_light_enum)

exit(gen.generate(PACKAGE, "two_d_guidance", "trr_world"))
